openlibrary/core/vendors.py

```diff
@@ -83,6 +84,15 @@ def __init__(self, key, secret, tag, host='webservices.amazon.com',
             host=host,
             region=region)
 
+    def search(self, keywords):
```

Comment:
Add comment that this is only used via CLI for testing; not used in the code anywhere

```suggestion
    def search(self, keywords):
```